### PR TITLE
Added typewriter animation

### PR DESCRIPTION
**Pull request by Naufil-Shemle**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  box-sizing: border-box;
+}
+
+:root {
+  --bg-color: hsl(49 37% 94%);
+  --typewriterSpeed: 6s;
+  --typewriterCharacters: 25;
+}
+
+body {
```
Comment (mmusk) [inline style.css:13]: group similar properties @Naufil-Shemle

**Review by mmusk — APPROVED** [review]
LGTM

File: style.css (hunk 1 above)
Comment (Naufil-Shemle) [reply style.css:13]: Done @mmusk

Comment (mmusk) [reply style.css:13]: LG

File: style.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  min-height: 100vh;
+
+  display: grid;
+
+  place-content: center;
+  text-align: center;
```
Comment (faran4engg) [inline style.css:18]: group them as well

**Review by faran4engg — APPROVED** [review]
LGTM

**Comment by faran4engg** [comment]
Also you can add live link in the About section @Naufil-Shemle 
Screenshot 2022-01-07 at 6 51 04 PM

File: style.css (hunk 2 above)
Comment (Naufil-Shemle) [reply style.css:18]: Done @faran4engg

**Review by Rehmatulla-Khan — APPROVED** [review]

**Review by mujeeb100 — COMMENTED** [review]
fabulous

**Review by Rehan-Khan-Engineer — APPROVED** [review]

**Review by nehalskdev — APPROVED** [review]

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]

**Pull request merged**